modules/apps/transfer/types/transfer_authorization_test.go

```diff
@@ -389,6 +389,84 @@ func (suite *TypesTestSuite) TestTransferAuthorizationAccept() {
 				suite.Require().Nil(res.Updated)
 			},
 		},
+		{
+			"success - allowed forwarding hops",
+			func() {
+				msgTransfer.Forwarding = types.NewForwarding("", types.Hop{PortId: "1", ChannelId: "channel-1"}, types.Hop{PortId: "2", ChannelId: "channel-2"})
```

Comment:
nit: would it make sense to use the same port as the other tests for consistency `ibctesting.MockPort`?

Reply:
Yeah good idea, done :)